crates/matrix-sdk-ui/src/timeline/error.rs

```diff
@@ -79,3 +79,26 @@ enum UnsupportedReplyItemInner {
     #[error("redacted events whose JSON form isn't available can't be replied")]
     MissingJson,
 }
+
+#[derive(Error)]
+#[error("{0}")]
+pub struct UnsupportedEditItem(UnsupportedEditItemInner);
```

Comment:
What's the wrapper for?

Reply:
Hiding what variants there are on the "actual" error type, so it's not a breaking change to rename or remove one. There's no reason users of this API would check for the error variant, it only exists as an implementation detail so we can print meaningful messages to logs.